planning/behavior_velocity_planner/src/node.cpp

```diff
@@ -203,35 +203,46 @@ BehaviorVelocityPlannerNode::BehaviorVelocityPlannerNode(const rclcpp::NodeOptio
 }
 
 // NOTE: argument planner_data must not be referenced for multithreading
-bool BehaviorVelocityPlannerNode::isDataReady(const PlannerData planner_data) const
+bool BehaviorVelocityPlannerNode::isDataReady(
+  const PlannerData planner_data, rclcpp::Clock clock) const
 {
   const auto & d = planner_data;
 
   // from tf
   if (d.current_pose.header.frame_id == "") {
+    RCLCPP_INFO(get_logger(), "Frame id of current pose is missing");
```

Comment:
@takayuki5168 
Can you also change this to throttle? to find out missing topic easier

Reply:
@takayuki5168 
ping

Reply:
pong

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1477/commits/af6157f9e7d449828f725ab840a8f461b16c844f
DONE